HowManyCharactersInTxtFile/src/main/java/org/Marta/FileUtil.java

```diff
@@ -0,0 +1,23 @@
+package org.Marta;
+
+import java.io.*;
+
+public class FileUtil {
+    static BufferedReader reader = null;
+
+    public FileUtil (String filePath) throws FileNotFoundException {
+        File file = new File(filePath);
+        FileInputStream fileInputStream = new FileInputStream(file);
+        InputStreamReader inputStreamReader = new InputStreamReader(fileInputStream);
+        reader = new BufferedReader(inputStreamReader);
+    }
+
+    public int countCharacters() throws IOException {
+        int numOfCharacters = 0;
+        String data;
+        while ((data = reader.readLine()) != null) {
+            numOfCharacters += data.length();
+        }
```

Comment:
nigdzie nie zamykasz tych otwartych strumieni. W twoim przypadku wydaje mi się, że wystarczy zamknąć BufferedReader . (close()). Z racji, że tworzysz to w konstruktorze to może warto zaimplementować interfejs https://docs.oracle.com/javase/8/docs/api/java/io/Closeable.html

a następnie w main() użyć konstrukcji "try with resources"  https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html